content/using/install.md

```diff
@@ -149,40 +159,34 @@ Click that `Chat` type on your Landscape home page, and then click the `Join Cha
 
 You can send and receive messages here, just like any other chatroom.
 
-### Make a Note
-
-Navigate back to your Landscape homepage. Typing `localhost` (or the correct equivalent) into your search bar does the trick.
-
-Then click the `Publish` tile. Fill the fields as appropriate to make a new Notebook and invite anyone you want to be able to read it as a subscriber. A Notebook can be treated like a blogs, or a public list of notes. One you have made a Notebook, you can start posting entries to it. Navigate back to it at any time by clicking the tile.
-
-## Planet Setup {#planet}
-
-If you have a planet, you have a full and persistent peer on the Urbit network.
-
-### Getting Urbit ID
+## Getting an Urbit ID (Planet or Star)
 
 Tlon doesn't currently sell or distribute Urbit IDs, but there are a few ways to get your own:
 
 - Getting an invitation from a friend
 - Purchasing an Urbit ID from a third party, such as [urbit.live](https://urbit.live), [OpenSea](https://opensea.io), [planet.market](https://planet.market/), or [Urbit Marketplace](https://urbitmarketplace.com/).
 
+Note that if you buy a planet from opensea you may have to change its star, ask around in the `urbit-community` chat as a comet first for help before purchasing.
```

Comment:
Should be noted here that this possibility of having to escape comes with buying a planet from _anyone_ not just opensea. You're more likely to have a problem there as it's not an urbit specific market, but it should be made clear this is not something specific to opensea.